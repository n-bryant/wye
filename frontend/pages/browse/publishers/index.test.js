import { PublishersPage } from "./index";

describe("PublishersPage", () => {
  const props = {
    calsses: {
      root: "root"
    }
  };

  it("should render successfully", () => {
    const wrapper = shallow(<PublishersPage {...props} />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
