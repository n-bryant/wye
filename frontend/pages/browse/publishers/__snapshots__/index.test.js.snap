// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PublishersPage should render successfully 1`] = `
<div
  className="wye-PublishersPage"
>
  <WithWidth(WithStyles(BrowseByPublisher)) />
</div>
`;
